src/utils/sgDbProjects.ts

```diff
@@ -244,7 +262,23 @@ export async function tryResolveMetadata({
         logoUri: null,
         tags: null,
         archived: null,
-      } as Json<DBProject>,
+      },
+    }
+  }
+
+  // If metadataUri has not changed, we don't need to resolve new metadata and can simply append existing dbProject metadata properties
+  if (sgProject.metadataUri === dbProject.metadataUri) {
```

Comment:
performance upgrade: we no longer resolve metadata for a project whose `metadataUri` hasn't changed (we previously would if _any_ property had changed)